src/routes/plans/[id]/+page.svelte

```diff
@@ -323,6 +323,9 @@
         <PlayIcon />
         <svelte:fragment slot="metadata">
           <div>Simulation Dataset ID: {$simulationDatasetId}</div>
+          <div>
+            Simulation Extent: {$simulationStatus === Status.Incomplete ? $simulationDataset?.extent : 'Finished'}
```

Comment:
Could there perhaps be a better user-facing name for this? "Simulation Extent" is a little ambiguous to me if we're trying to communicate simulation progress / elapsed time / status?